authors/apps/authentication/tests/test_password_reset.py

```diff
@@ -1,21 +1,156 @@
+import json
 from django.test import TestCase
-from rest_framework import test, status
 from django.urls import reverse
-from ..models import User
+from rest_framework import test, status
+from ..models import User, ResetPasswordToken
+
 
 
-class PasswordResetTest(TestCase):
+class TestPasswordResetRequest(TestCase):
+    """
+    tests password reset functionality
+    """
     def setUp(self):
-        pass
+        self.client = test.APIClient()
+        self.user = User.objects.create(
+            email="Redhawk@gmail.com",
+            password="fireandblood",
+            username="doggetofthehill"
+        )
+        self.user_email = {"email": self.user.email}
```

Comment:
Most of these variables set here are only being used once hence beats the purpose of adding them as instance variables. I'm of the opinion that setting them as instance variables isn't necessary unless you plan on adding more tests that will reuse them

Reply:
I have edited the tests to reflect these changes